kit/service/transport.go

```diff
@@ -131,7 +132,10 @@ func (r *Router) MountRoute(route Route) *Router {
 		_, handler = newrelic.WrapHandle(r.newrelicApp, route.GetPath(), handler)
 	}
 
-	r.prefixMux.Methods(route.GetMethod()).Path(route.GetPath()).Handler(handler)
+	headers := handlers.AllowedHeaders([]string{"X-Connect-Client-Type", "X-Connect-Client-Version", "Origin", "Content-Type"})
```

Comment:
No its not removed. The AllowedHeaders method performs an append https://github.com/gorilla/handlers/blob/master/cors.go#L168